CMSgov/mint-app
# [EASI-3551] Add new payment type to claims based payments

# EASI-3551

## Changes and Description

- Add new payment type to gql and data model



## How to test this change

1. Navigate to Claims-based payments in the payment section and select `Payments for post-discharge home visits` from the dropdown
2. Ensure option is saved on the task list and on the read only page



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
